### PR TITLE
Restoring '!!service.getQueue' in ServicesTable.js

### PR DESCRIPTION
**Pull request by wavesoft**
~Boy-scout clean-up went wrong :rotating_light:~
Boy-scout clean-up done right 👮

### CODE REVIEW

File: plugins/services/src/js/containers/services/ServicesTable.js [hunk 1]
```diff
@@ -232,7 +232,11 @@ class ServicesTable extends React.Component {
       verboseOverview = ` (${tasksRunning} of ${instancesCount} Instances)`;
     }
 
-    let queue = service.getQueue();
+    let queue = null;
+    if (!!service.getQueue) {
```
Comment (orlandohohmeier) [inline plugins/services/src/js/containers/services/ServicesTable.js]: Let's add `getQueue` to the service struct instead of doing this check...

Comment (jfurrow) [inline plugins/services/src/js/containers/services/ServicesTable.js]: We'll also need to add it to `ServiceTree` if we go down this route.

**Comment by jfurrow** [comment]
This fixes the bug and LGTM after @orlandohohmeier's comment is addressed! :ship:

File: plugins/services/src/js/containers/services/ServicesTable.js (hunk 1 above)
Comment (wavesoft) [inline plugins/services/src/js/containers/services/ServicesTable.js]: Ok, that sounds like a better 'clean-up' will do 👍

**Comment by rcorral** [comment]
LGTM! Thanks!

**Comment by orlandohohmeier** [comment]
LGTM 💯 ✨

**Pull request merged**